README.md

```diff
@@ -71,3 +71,19 @@ Adding a new scheme is simple:
     Sanscript.addRomanScheme(schemeName, schemeData);
 
 For help in creating `schemeData`, see the comments on the `addBrahmicScheme` and `addRomanScheme` functions.
+
+Testing
+-------
+
+Top level `Makefile` Starts a Webserver using `PHP` on Port `9732`.
+
+`make` - will start the webserver on port `9732`
+
+Open your browser and launch this url `http://127.0.0.1:9732/test/index.html` and see that all tests are ok.
```

Comment:
I don't see why this is necessary. Can you explain? The `localhost`/qunit approach that exists already seems fine.